### PR TITLE
Add modal to look up BGS participant ID

### PR DESCRIPTION
**Pull request by lowellrex**
I spent a few hours today helping determine why a private bar attorney saw the cases that they saw. The reason ended up being that the BGS participant ID for their organization was for the a different organization than the one they expected. This tool allows folks who manage Caseflow teams determine the correct BGS participant ID without the intervention of developers.

Successful lookup | Lookup with error
--- | ---
happy-path | error

### CODE REVIEW

**Comment by hschallhorn** [comment]
Not seeing alerts/warnings inside of the modal.
Screen Shot 2019-08-16 at 9 56 05 AM

File: client/app/queue/LookupParticipantIdModal.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      then((resp) => this.setState({ representedOrganizations: resp.body.represented_organizations })).
+      catch((err) => this.props.showErrorMessage({ title: 'Error',
+        detail: err.message }));
+  }
+
+  representedOrganizationsList = () => <ol>
+    { this.state.representedOrganizations.map((org, idx) =>
+      <li key={idx}>{org.representative_name}, {org.representative_type}. Participant ID: {org.participant_id}</li>
+    ) }
+  </ol>;
```
Comment (hschallhorn) [inline client/app/queue/LookupParticipantIdModal.jsx]: Can we add a quick check for if `representedOrganizations` is empty here to show some sort of "User does not represent any VSOs" message?

Comment (lowellrex) [reply client/app/queue/LookupParticipantIdModal.jsx]: Added!

image

**Comment by lowellrex** [comment]
> Not seeing alerts/warnings inside of the modal.

Added!

image

**Pull request merged**